less/footer.less

```diff
@@ -1 +1,16 @@
-// Footer Styles
\ No newline at end of file
+// Footer Styles
+footer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: @footer-bg-color;
+    color: @footer-font-color;
+    height: 80px;
+    border: 1px solid @footer-border-color;
+    border-radius: 5px;
+    padding-top: 10px;
+    font-size: 1.6rem;
```

Comment:
A good idea is to set common font-sizes in variables.less, so when you need to change the font sizes, you can do so in one file, instead of searching around.